docs/finding-records.md

```diff
@@ -0,0 +1,15 @@
+Find Records
+==============================================================================
+
+```js
+return this.store.query('post', {
+  include: 'user,comments',
```

Comment:
I wonder if this would be better as an array

Reply:
Ah, this is the standard ED syntax, isn't it?

Reply:
Yeah it is. I'm actually not in love with this solution because it's prob preferable to just pass in the selected columns the normal way, through the filter function through `select()`. I just put a little convenience function in the adapter that converts that include string to the  `* ()` select format Supabase/PostgREST wants.